flycheck.el

```diff
@@ -9742,24 +9742,40 @@ See URL `https://www.veripool.org/wiki/verilator'."
           line ": " (message) line-end))
   :modes verilog-mode)
 
+(flycheck-def-option-var flycheck-xml-xmlstarlet-xsd-path nil xml-xmlstarlet
+  "An XSD schema to validate against."
+  :type '(file :tag "XSD schema")
+  :safe #'stringp
+  :package-version '(flycheck . "32"))
```

Comment:
We are in `31-snapshot`, so that means 31 has not been released yet.  (In doubt, you can also check the release tags and see that the latest one is 30).